c10/xpu/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Build file for the C10 XPU.
+#
+# C10 XPU is a minimal library, but it does depend on SYCL.
+
+include(../../cmake/public/xpu.cmake)
+
+# ---[ Configure macro file.
+set(C10_XPU_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS}) # used in cmake_macros.h.in
```

Comment:
Why this is needed?

Reply:
This option is used to build for windows. Let me remove it.